CONTRIBUTE.MD

```diff
@@ -16,13 +16,5 @@ We strongly suggest asking for feedback before starting to work on a new feature
 functionality, etc. so that we can provide feedback and discuss how we can 
 collaborate on this and support contributors. 
 
-To engage with project maintainers, contributors can reach out on 
-[Zulip](https://openvmm.zulipchat.com/) and then start drafting a 
-[GitHub issue](https://github.com/microsoft/openvmm/issues/new?template=Blank+issue) 
-after getting some preliminary feedback on Zulip, or they can open a GitHub issue 
-to get preliminary feedback on GitHub and then reach out to us on Zulip to get 
-some more synchronous support. 
-
-We normally use Zulip for more informal conversations and GitHub Issues for more 
-formal, auditable, public discussion and progress tracking. We encourage contributors 
-to use both in either order based on preference.
+To engage with project maintainers, contributors can reach out through 
+[GitHub issues](https://github.com/microsoft/openvmm/issues/new?template=Blank+issue).
```

Comment:
Can we update this to point to the discussions page instead?
https://github.com/microsoft/openvmm/discussions